ComSemApp/student/views.py

```diff
@@ -798,7 +798,10 @@ def get_context_data(self, **kwargs) -> dict[str, dict[str, str | int | float]]:
         """
         context : dict[str, Any] = super(SpeakingPracticeResultsView, self).get_context_data(**kwargs)
         context['attempts'] : list[dict[str, str | float]] = {}
-        attempt_ids : list[str] = dict(self.request.GET)['attempt']
+        try:
+            attempt_ids : list[str] = dict(self.request.GET)['attempt']
```

Comment:
I like this try catch, but I feel as though there should be a guard client-side as well to prevent empty requests being sent by the button in the first place

Reply:
I don't think a client-side guard can be made unless we want to disable the results button until atleast one successful attempt id is retrieved on the problem page. I might just have a misunderstanding of the GET request form works though

Reply:
I'm going to make this a new issue so that I can go ahead and merge these fixes into Placeholder Pages

Reply:
#208 created to address this issue at a later time.